.github/workflows/build.yml

```diff
@@ -0,0 +1,22 @@
+name: Build Website
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy-website:
+    name: Build Website
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: '2.7.0'
+        bundler-cache: true
+    - name: Run .github_actions.sh
```

Comment:
I suggest running the two commands directly from here (and removing the file .github_actions.sh)

Reply:
That's fair. I'll change it right away.

Reply:
This name doesn't mean anything because that file doesn't exist.  Please change it to something that is more descriptive.